config-hbase.sh

```diff
@@ -3,7 +3,7 @@
 # . /build/config.sh
 
 HBASE_DIST="http://apache.cs.utah.edu/hbase"
-#HBASE_DIST="http://archive.apache.org/dist/hbase"
+HBASE_DIST="http://archive.apache.org/dist/hbase"
```

Comment:
Not sure what you'd like me to do here. The current version no longer exists on the live mirror.